spyder/config/main.py

```diff
@@ -352,7 +352,7 @@
               '_/switch to explorer': "Ctrl+Shift+X",
               '_/switch to plots': "Ctrl+Shift+G",
               # -- In widgets/findreplace.py
-              '_/find text': "Ctrl+F",
+              'find_replace/find text': "Ctrl+F",
               '_/find next': "F3",

```

Comment:
I think it's better if we change the context of all shortcuts defined in widgets/findreplace.py to be `find_replace`.

Reply:
Agreed!